src/FMBot.Bot/Services/SettingService.cs

```diff
@@ -324,6 +324,43 @@ public SettingService(IDbContextFactory<FMBotDbContext> contextFactory)
         return settingsModel;
     }
 
+    public static DiscogsCollectionSettings SetDiscogsCollectionSettings(string extraOptions = null)
+    {
+        var collectionSettings = new DiscogsCollectionSettings
+        {
+            Formats = new List<DiscogsFormat>(),
+            NewSearchValue = extraOptions
+        };
+        
+        if (extraOptions == null)
+        {
+            return collectionSettings;
+        }
+
+        var vinylFormats = new[] { "format:vinyl", "f:vinyl" };
```

Comment:
maybe we can get rid of the `format:` / `f:` parameter for now. Because the format words are not words which are generally used in artist names or album names

Reply:
that's generally true. i suppose as a minimum viable product we can hope no one wants to query artists containing those keywords, haha